test/unit_tests/location_service/set_distance_between_user_and_games_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:niira/models/location.dart';
+import 'package:niira/services/location_service.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:niira/utilities/calc_distance.dart';
+import 'package:test/test.dart';
+
+import '../../mocks/data/mock_games.dart';
+
+void main() {
+  test(
+      'locationService.setDistanceBetweenUserAndGames returns list of games with distanceFromUser set',
+      () {
+    // init mock service
+    final geolocator = GeolocatorPlatform.instance;
+    final locationService = LocationService(geolocator);
+    // init mock data
+    final mockUserLocation =
+        Location(latitude: -37.865351, longitude: 144.989012);
+    final mockGames = MockGames().gamesWithoutDistance;
+
+    // generate expected outcome
```

Comment:
just wanna say the tests are very readable because of the high quality comments, thanks!

Reply:
thank you really appreciate it nick!